seed_services_cli/identity_store.py

```diff
@@ -110,3 +116,32 @@ def identities_from_json(json_file):
             raise click.UsageError(
                 "JSON file lines must be objects.")
         yield data
+
+
+@click.option(
+    '--json-file', type=click.File(file_open_mode),
+    help=('JSON objects, details that will be updated'))
+@click.pass_context
+def identities_details_update(ctx, json_file):
+    """ Update identities details fields.
+    """
+    if not json_file:
+        raise click.UsageError("Please specify --json_file.")
+    api = get_api_client(ctx.obj.identity_store.api_url,
+                         ctx.obj.identity_store.token)
+
+    update_data = json.loads(json_file.read().rstrip("\n"))
+
+    for key in update_data:
```

Comment:
We can do `for key, patch in update_data.items()` here to avoid having to do `update_data[key]` later on